descriptors/ModuleDescriptor-template.json

```diff
@@ -998,6 +998,22 @@
         }
       ]
     },
+    {
+      "id": "orders.routing-list",
+      "version": "1.0",
+      "handlers": [
+        {
+          "methods": ["GET"],
+          "pathPattern": "/orders/routing-lists/{id}/template",
```

Comment:
It sounds like just getting template stored in mod-template-engine.
What do you think about POST /orders/routing-lists/{id}/process-template ?

Reply:
First I thought in that way, the next, since api is no longer doing modification operations, get make more sense to me. 

POST is not [idempotent ](https://restfulapi.net/idempotent-rest-apis/) method, but our api is doing work that is idempotenent. That is mean, multiple request have same effect and don't change the db records.

regarding api -> user shouldn't need to know what should be done to get template, so /template this name come to mind. more like hiding internar dependencies.

Overall, for me, api tell the user enter routing list id and get required template.